inception-project/inception
# #1760 - Getting started

**What's in the PR**
* We update the getting started guide

**How to test manually**
* Run `de/tudarmstadt/ukp/inception/doc/GenerateDocumentation.java` and check 
  the outpu in `inception/target/doc-out`

**Automatic testing**

**Documentation**
